### ISSUE TITLE
Feature Request: Instructions on Focus Chart page

### ISSUE THREAD

**Issue by evilC** [hd-zero/hdzero-goggle#250]
When loaning your goggles to someone to try them out, you would generally put the goggles onto the Focus Chart page, as that is the first thing they will need to do.  
Why not put some instructions on this page? For example, some text pointing to where the focus/IPD knobs are (And how to use them), plus some text pointing to the top left saying how to exit out of the menu (eg "Long press button HERE twice to exit to FPV view")

**Comment by AlexandreBonneau** [comment]
![Focus chart HDZero r1](https://github.com/hd-zero/hdzero-goggle/assets/7248877/eb2d7c79-2bd2-4d77-88b6-ddeff7a8bcfd)
Adding instructions to change the IPD is hard on the existing focus chart, since there is not much space left.
Also, if you lend your goggles to somebody, you can just explain that to them.

I've added some filler text all around the focus chart because while having the circles is handy,I find it important to get text that you ought to be able to read everywhere on the screen as well.

**Comment by evilC** [comment]
> Adding instructions to change the IPD is hard on the existing focus chart, since there is not much space left. Also, if you lend your goggles to somebody, you can just explain that to them.

Agreed, not that big of a deal as they are in a pretty standard place.  
Looks good!

**Comment by ligenxxxx** [comment]
TODO
